plugins/woocommerce/includes/class-wc-countries.php

```diff
@@ -1433,11 +1433,17 @@ public function get_country_locale() {
 					),
 					'VN' => array(
 						'state'     => array(
-							'required' => false,
-							'hidden'   => true,
+							'priorty'  => 70,
+							'label'    => __( 'Province/City', 'woocommerce' ),
+							'required' => true,
+							'hidden'   => false,
```

Comment:
I would remove this `hidden` and `required` config. By default, `hidden` has already been `false`.

For `required`:
https://github.com/woocommerce/woocommerce/blob/018ce13b49bfa8a0f7653b5c2bd4c11dbaffd322/plugins/woocommerce/includes/class-wc-countries.php#L741